[BUG] If I keep idle the app for some times, app is not refreshed; Hence schedule can be create at previous time

**Bug Description** When the app is left idle for a while, it does not refresh. As a result, schedules can be created for past times.

**Steps to Reproduce**

1. Visit the app and note the current time (e.g., 2:30 PM).
2. Wait for 30 minutes until the time is now 3:00 PM.
3. Without reloading the page, click on the "Book Now" button for a 15-minute slot.
4. A schedule is created for the previous time slot (2:30–2:45 PM).

**Expected Behavior**
The app should not allow the creation of a meeting schedule for a past time. Instead, it should display an alert or toast message and refresh to update the current time.

**Screenshots**
If applicable, add screenshots to help explain your problem.
![image](https://github.com/user-attachments/assets/3912f6b3-ab7e-4a51-84aa-a906c745eb96)

> There is some issue with the setup. The client side is not running
> ```
> Environment variable react-scripts start not defined
> npm ERR! Lifecycle script `env` failed with error:
> npm ERR! Error: command failed
> npm ERR!   in workspace: @bookify/client@0.1.0
> npm ERR!   at location: C:\Users\naman\Desktop\OSC\bookify\client
> ```

Please read the documentation. There shouldn't be anything wrong with the setup.

--- a/client/src/pages/Home/BookRoomView/index.tsx
+++ b/client/src/pages/Home/BookRoomView/index.tsx
@@ -3,7 +3,7 @@ import { useEffect, useRef, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import Dropdown, { DropdownOption } from '../../../components/Dropdown';
 import LoadingButton from '@mui/lab/LoadingButton';
-import { Action, convertToRFC3339, createDropdownOptions, getTimeZoneString, isChromeExt, renderError } from '../../../helpers/utility';
+import { Action, convertToRFC3339, createDropdownOptions, getTimeZoneString, isChromeExt, populateTimeOptions, renderError } from '../../../helpers/utility';
 import toast from 'react-hot-toast';
 import AccessTimeFilledRoundedIcon from '@mui/icons-material/AccessTimeFilledRounded';
 import PeopleRoundedIcon from '@mui/icons-material/PeopleRounded';
@@ -163,6 +163,20 @@ export default function BookRoomView({ onAction, refresh, setRefresh }: BookRoom
 
     const date = new Date(Date.now()).toISOString().split('T')[0];
     const formattedStartTime = convertToRFC3339(date, startTime);
+
+    if (new Date() > new Date(formattedStartTime)) {
+      toast.error(`Time mismatch. Try again`);
+      setLoading(false);
+      const options = createDropdownOptions(populateTimeOptions());
+      setTimeOptions(options);
+      setFormData((prev) => {
+        return {
+          ...prev,
+          startTime: options[0].value,
+        };
+      });
+      return;
+    }
 
     const floor = await cacheService.get('floor');
     const preferredTitle = (await cacheService.get('title')) || undefined;

